.github/workflows/ql-for-ql-build.yml

```
@@ -137,20 +137,20 @@ jobs:
         env:
           CONF: ./ql-for-ql-config.yml
       - name: Initialize CodeQL
-        uses: github/codeql-action/init@77a8d2d10c0b403a8b4aadbd223dc489ecd22683
+        uses: github/codeql-action/init@45955cb1830b640e2c1603ad72ad542a49d47b96
         with:
           languages: ql
           db-location: ${{ runner.temp }}/db
           config-file: ./ql-for-ql-config.yml
           tools: ${{ steps.find-latest-bundle.outputs.url }}
       - name: Move pack cache
```

Suggest a slight renaming to keep consistency, but otherwise LGTM.

```suggestion
      - name: Move pack queries
```